### PR TITLE
Triplet Family sol. in py for GFG PODT[27 oct 24]

### PR DESCRIPTION
**Pull request by thedarkking01**
@Ayu-hack  Hii 😊
today problem solved in python 
pic : 
image
image

### CODE REVIEW

File: October 2024/Triplet Family[ 27 oct].py
Comment (Ayu-hack) [inline October 2024/Triplet Family[ 27 oct].py:1]: great work buddy!

**Comment by Ayu-hack** [comment]
Great Work!
 Merged.
Thank you for contributing .
Feel free to tag me in your Hacktoberfest certificate or badges LinkedIn post! 😄 
👉 Follow my GitHub and connect with me on [LinkedIn](https://www.linkedin.com/in/ayush-tiwari02)!

**Comment by Ayu-hack** [comment]
if possible do follow on [GitHub](https://github.com/Ayu-hack) and connect on [LinkedIn](https://www.linkedin.com/in/ayush-tiwari02) as a token of appreciation for the efforts! 🙌

File: October 2024/Triplet Family[ 27 oct].py
Comment (thedarkking01) [reply October 2024/Triplet Family[ 27 oct].py:1]: Thanks

**Comment by thedarkking01** [comment]
> if possible do follow on [GitHub](https://github.com/Ayu-hack) and connect on [LinkedIn](https://www.linkedin.com/in/ayush-tiwari02) as a token of appreciation for the efforts! 🙌

i sent you linkedin request by sourabh singh bais

**Comment by thedarkking01** [comment]
@Ayu-hack  Hii 
you didn't merge it with github can you please check it

**Pull request merged**